src/utils/net.rs

```diff
@@ -44,19 +43,18 @@ pub async fn reader(location: &str) -> anyhow::Result<impl AsyncRead> {
             (Left(stream), content_length)
         }
         Err(_) => {
-            info!("reading file: {}", location);
+            info!("Reading file: {}", location);
             let stream = tokio::fs::File::open(location).await?;
             let content_length = stream.metadata().await?.len();
             (Right(stream), content_length)
         }
     };
 
-    if content_length > 0 {
-        read_progress.set_length(content_length);
-        read_progress.set_style(progress_style())
-    }
-
-    let mut reader = tokio::io::BufReader::new(read_progress.wrap_async_read(stream));
+    let mut reader = tokio::io::BufReader::new(ProgressLog::wrap_async_read(
+        "Loading",
+        stream,
+        content_length,
+    ));
```

Comment:
I don't think this is the right place for this - _callers_ of `reader` should decide if their call is worth logging or not

(I'm aware I touched this code :D)

Reply:
Hmmm. With the new progress logger design, if the call is not worth logging (< 5s) you won't see anything in the console.